charts/nri-metadata-injection/values.yaml

```diff
@@ -1,81 +1,94 @@
+# -- Annotations to be added to all pods created by the integration.
+podAnnotations: {}
+# -- Additional labels for chart pods. Can be configured also with `global.podLabels`
+podLabels: {}
+
+# -- Image for creating the needed certificates of this webhook to work
+# @default -- 100m/30M -/80M
 resources:
   limits:
     memory: 80M
   requests:
     cpu: 100m
     memory: 30M
 
-# Configure podSecurityContext
-podSecurityContext:
+# -- Sets pod's priorityClassName. Can be configured also with `global.priorityClassName`
+priorityClassName: ""
+# -- (bool) Sets pod's hostNetwork. Can be configured also with `global.hostNetwork`
+# @default -- false
+hostNetwork:
+# -- Sets pod's dnsConfig. Can be configured also with `global.dnsConfig`
+dnsConfig: {}
+# -- Sets security context (at pod level). Can be configured also with `global.podSecurityContext`
+podSecurityContext: {}
+# -- Sets security context (at container level). Can be configured also with `global.containerSecurityContext`
+containerSecurityContext: {}
 
-# Configure containerSecurityContext
-containerSecurityContext:
+certManager:
+  # certManager.enabled -- Use cert manager for webhook certs
+  enabled: false
 
-# Use custom tls certificates for the webhook, or let the chart handle it
-# automatically.
-#
-# Make sure to read the 'Manage custom certificates' section of the official
-# docs when using custom certificates.
-#
-# https://docs.newrelic.com/docs/integrations/kubernetes-integration/link-your-applications/link-your-applications-kubernetes#configure-injection
-customTLSCertificate: false
+# -- Sets pod/node affinities. Can be configured also with `global.affinity`
+affinity: {}
+# -- Sets pod's node selector. Can be configured also with `global.nodeSelector`
+nodeSelector: {}
+# -- Sets pod's tolerations to node taints. Can be configured also with `global.tolerations`
+tolerations: []
 
-# If you wish to provide additional annotations to apply to the pod(s), specify
-# them here
-# podAnnotations:
+# -- Enable the metadata decoration only for pods living in namespaces labeled
+# with 'newrelic-metadata-injection=enabled'.
+injectOnlyLabeledNamespaces: false
 
-# Pod scheduling priority
-# Ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
-# priorityClassName: high-priority
+# -- Use custom tls certificates for the webhook, or let the chart handle it
+# automatically.
+# Ref: https://docs.newrelic.com/docs/integrations/kubernetes-integration/link-your-applications/link-your-applications-kubernetes#configure-injection
+customTLSCertificate: false
 
-# Webhook timeout
-#
-# Configure how long the API server should wait for a webhook to respond before treating the call as a failure
-#
+# -- Webhook timeout
 # Ref: https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#timeouts
-timeoutSeconds: 30
-
-# Use cert manager for webhook certs
-certManager:
-  enabled: false
-
-tolerations: []
+timeoutSeconds: 28
```

Comment:
That was due to an issue, right? Can we add a link into the code?

Reply:
It is already. The `Ref` link goes to a doc that says that the value should between 0 and 30 (not included)